packages/cicero-core/src/templateloader.js

```diff
@@ -52,6 +52,7 @@ class TemplateLoader extends FileLoader {
         const sampleTextFiles = {};
 
         const readmeContents = await TemplateLoader.loadZipFileContents(zip, 'README.md');
+        const logo = await TemplateLoader.loadZipFileContents(zip, 'logo.png');
```

Comment:
This will load the logo file into a string (and perform normalisation). I think we will need to extend the FileLoader to add a method to load data into a Buffer.

https://github.com/accordproject/ergo/blob/master/packages/ergo-compiler/lib/fileloader.js#L57

Reply:
E.g. `loadZipFileBuffer`